lightning/src/util/ser_macros.rs

```diff
@@ -855,6 +858,7 @@ macro_rules! impl_writeable_tlv_based {
 ///
 /// [`Readable`]: crate::util::ser::Readable
 /// [`Writeable`]: crate::util::ser::Writeable
+#[macro_export]
 macro_rules! tlv_stream {
```

Comment:
Note that this and the below macros generate code specifically needed for `offers` messages. I don't think you would need these.